microshift_install_get_ready/microshift-install-get-ready.adoc

```diff
@@ -16,3 +16,70 @@ include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
 Plan to pair a supported version of {op-system-ostree} with the {microshift-short} version you are using as described in the following compatibility table:
 
 include::snippets/microshift-rhde-compatibility-table-snip.adoc[leveloffset=+1]
+
+[id="get-ready-install-tools-intro_{context}"]
+== Installation tools for {microshift-short} installation types
```

Comment:
We mention "installation types" in the heading, but did not yet explain them. Lets do this here (see other comments below)